### ISSUE TITLE
Sandstorm.io compatibility

### ISSUE THREAD

**Issue by rodrigok** [RocketChat/Rocket.Chat.Cordova#30]
As mentioned here https://github.com/RocketChat/Rocket.Chat.Cordova/issues/26#issuecomment-159681223 we need to test and fix problems with Sandstorm.io connection.
##

**Comment by pdehaye** [comment]
I just tested with Cordova Beta 8 version 14487156..., and a fresh install of a Sandstorm grain (announced version 0.6, in any case after RocketChat/Rocket.Chat/#1526 was fixed). It still does not work, but I am not entirely sure I understood the process. I generated a webkey from the grain, and pasted that into my Cordova testing iPhone app. I get the error

```
Failed to connect to server: error. Forbidden. 
```

@paulproteus ' comments on #26 are probably pointing in the right direction.

**Comment by geekgonecrazy** [comment]
Does #32 solve this?

**Comment by pdehaye** [comment]
I don't think so, #32 led to beta 6.

**Comment by geekgonecrazy** [comment]
@pdehaye right, so that fixed part of the issue.  I'll give this a try and see what I come up with.

**Comment by geekgonecrazy** [comment]
#114 will close this.
